internal/icinga2/api_responses.go

```diff
@@ -80,6 +81,14 @@ type CheckResult struct {
 	ExecutionEnd   UnixFloat `json:"execution_end"`
 }
 
+// MarshalLogObject implements the zapcore.ObjectMarshaler interface.
+func (cr *CheckResult) MarshalLogObject(encoder zapcore.ObjectEncoder) error {
+	encoder.AddInt("exit_status", cr.ExitStatus)
+	encoder.AddInt("state", cr.State)
+	encoder.AddString("output", cr.Output)
```

Comment:
The output might be very long and might even contain sensitive data, that should not be logged. Otherwise, without this field the log message might be next to useless.

Thus, as long as it is only used for debug logging, this should be fine. And it is currently being logged anyway.

The same applies for the other types.